olmo/train.py

```diff
@@ -82,6 +95,11 @@ def reset(self) -> None:
 
     def check(self) -> Dict[str, float]:
         metrics: Dict[str, float] = {"throughput/total_tokens": self.global_total_tokens}
+
+        # plot flops related metrics
+        metrics["throughput/total_training_Gflops"] = self.total_training_Gflops
+        metrics["throughput/total_training_log_Gflops"] = math.log(self.total_training_Gflops)
```

Comment:
Why not just `throughput/Gflops`?
```suggestion
        metrics["throughput/Gflops"] = self.total_training_Gflops
        metrics["throughput/log_Gflops"] = math.log(self.total_training_Gflops)
```

Reply:
I guess "total" is meant to convey that it's forward plus backward flops, but for other metrics we use "total" to mean "cumulative throughout training," like with `throughput/total_tokens`.

Reply:
yeah, so the idea is that training flops consumed until this step!

Reply:
Oh, but that would reset on every restart. What do you use this metric for?

Reply:
is there a ``global_batch_seen_until_now`` metric?

Reply:
I guess, I can do ``total_tokens`` / (``seq_len`` * ``global_batch_size``)

Reply:
ah, lets refactor ``num_fwf_flops`` and ``num_bck_flops`` to be per token and then a simple multiplication with ``global_total_tokens`` solves everything!